src/SGB_Command_Border.md

```diff
@@ -77,9 +82,15 @@ border defining all unique tiles would have to define this many tiles:
 -   (256\*224-160\*144)/64+1 = 537 tiles in fullscreen border
 -   (256\*176-160\*144)/64+2 = 346 tiles in widescreen border
 
-But the CHR RAM allocated by SGB for border holds only 256 tiles. This
-means a fullscreen border must repeat at least 281 tiles and a
-widescreen border at least 90.
+Because the CHR RAM allocated by SGB for border holds only 256 tiles,
+a fullscreen border must repeat at least 281 tiles and a widescreen
+border at least 90.
```

Comment:
```suggestion
Because the CHR RAM allocated by SGB for border holds only 256 tiles, a full-screen border must repeat at least 281 tiles and a widescreen border at least 90.
```
What does "widescreen" mean here, though?

Reply:
"Widescreen" means a border that is letterboxed for modern widescreen televisions. When the user activates the zoom mode on the television, the television crops the top and bottom of the signal. A game with more "modern" design sensibilities may deliberately not put anything in the top and bottom 24 pixels so that users of modern televisions don't miss anything.

Reply:
Do you think it'd be appropriate to describe this in a footnote?

Reply:
Yes. It depends on 1. what syntax to use to make footnotes in this Markdown variant, which is unfamiliar to me, and 2. in how much detail we want to describe [overscan](https://www.nesdev.org/wiki/Overscan).